tests/functional/aws-node-sdk/test/object/auth/tagConditionKeys.js

```diff
@@ -0,0 +1,58 @@
+const assert = require('assert');
+
+const withV4 = require('../../support/withV4');
+const BucketUtility = require('../../../lib/utility/bucket-util');
+const { makeTagQuery, updateRequestContexts } =
+    require('../../../../../../lib/api/apiUtils/authorization/tagConditionKeys');
+const { DummyRequestLogger, TaggingConfigTester, createRequestContext } = require('../../../../../unit/helpers');
+
+const taggingUtil = new TaggingConfigTester();
+const log = new DummyRequestLogger();
+const bucket = 'bucket2testconditionkeys';
+const object = 'object2testconditionkeys';
+const objPutTaggingReq = taggingUtil
+.createObjectTaggingRequest('PUT', bucket, object);
+const requestContexts = [createRequestContext('objectPutTagging', objPutTaggingReq)];
+
+describe('Tag condition keys updateRequestContext', () => {
+    withV4(sigCfg => {
+        let bucketUtil;
+        let s3;
+
+        beforeEach(() => {
+            bucketUtil = new BucketUtility('default', sigCfg);
+            s3 = bucketUtil.s3;
+            return s3.createBucketPromise({ Bucket: bucket })
+            .catch(err => {
+                process.stdout.write(`Error creating bucket: ${err}\n`);
+                throw err;
+            });
```

Comment:
Not sure how helpful catching an error just to log it is in the context of tests. Wouldn't this just alter the call stack for adding a log statement that we could probably already conclude from the error itself given line numbers and test names?

Reply:
yeah, this is another standard throughout our test suites